starter-code/index.html

```diff
@@ -3,40 +3,71 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel='stylesheet' href='css/main.css'>
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <nav>
+    <img src='images/spotify-logo.png' alt='spotify-logo' class='left'>
+    <ul class='right menu'>
+      <li>Premium</li>
+      <li>Discover</li>
+      <li>Help</li>
+      <li>Download</li>
+    </ul>
+  </nav>
+  <div class='clearfix'></div>
+  <div id='header'>
+    <div>
+    <h1>Music for everyone.</h1>
+    <h2>Spotify is now free on mobile, tablet and computer.
+        Listen to the right music, wherever you are.</h2>
+    </div>
+  </div>
+  <div>
+    <div class='whats-on'>
+    <h3>What’s on Spotify?</h3>
+    </div>
+    <div class='container'>
+      <ul>
+        <li class='left'>
+          <img src='images/music-icon.png'>
+          <h4 class='color-text'>Millions of Songs</h4>
+          <p>There are millions of
+          songs on Spotify</p>
+        </li>
+        <li class='center'>
+          <img src='images/high-quality-icon.png'>
+          <h4 class='color-text'>HD Music</h4>
+          <p>Listen to music as if you
+              were listening live</p>
+        </li>
+        <li class='right'>
+          <img src='images/devices-icon.png'>
+          <h4 class='color-text'>Stream Everywhere</h4>
+          <p>Stream music on your
+          smartphone, tablet or
+          computer</p>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class='clearfix'></div>
+  <div class='green-box'>
+    <div class='relative'>
+      <h3>It’s as yeezy as Kanye West.</h3>
+      <h4>Search</h4>
+      Know what you want to listen to?
+      Just search and hit play.
+      <h4>Browse</h4>
+      Check out the latest charts,
+      brand new releases and great
+      playlists for right now.
+      <h4>Discover</h4>
+      Enjoy new music every Monday
+      with your own personal playlist.
+      Or sit back and enjoy Radio.
+    </div>
+    <img src='images/spotify-app.jpg' class='right kayne' alt='app'>
```

Comment:
Casí lo tenías. Fíjate bien donde está colocado el img comparado con el div relative: está afuera en vez de dentro. Con colocarlo dentro y agregando un top: 50px (o lo que quisieras) se te solucionaba el problema!! Lo que hace el cansancio... nos pasa a todos 
🌵i got 99 problems but this img ain't one 🌵